ivy/functional/ivy/linear_algebra.py

```diff
@@ -758,21 +758,25 @@ def eigvalsh(
 
     >>> x = ivy.array([[[3.0,2.0],[2.0,3.0]]])
     >>> y = ivy.zeros([1,2])
-    >>> ivy.eigvalsh(x, out=y)
+    >>> ivy.eigvalsh(x,out=y)
     >>> print(y)
     ivy.array([[1., 5.]])
 
+    >>> x = ivy.array([[[4.0,2.0],[2.0,4.0]]])
+    >>> y = x.eigvalsh()
+    >>> print(y)
+    ivy.array([[2., 6.]])
+
     >>> x = ivy.array([[[3.0,2.0],[2.0,3.0]]])
     >>> ivy.eigvalsh(x, out=x)
-    >>> print(x)
     ivy.array([[1., 5.]])
 
     >>> x = ivy.array([[[2.0,3.0,6.0],[3.0,4.0,5.0],[6.0,5.0,9.0]],
     ...                [[1.0,1.0,1.0],[1.0,2.0,2.0],[1.0,2.0,2.0]]])
     >>> y = ivy.eigvalsh(x, UPLO="U")
     >>> print(y)
     ivy.array([[-1.45033181e+00,  1.02829754e+00,  1.54220343e+01],
-       [-1.12647155e-15,  4.38447177e-01,  4.56155300e+00]])
+       [-1.18198260e-15,  4.38447177e-01,  4.56155300e+00]])
```

Comment:
For me this value still seems to be `-1.12647155e-15`. Not sure why you might be getting a different value. Is your backend set to numpy? what version of np do you have in your env?